### ISSUE TITLE
Homepage accessibility: collection images

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#739]
### Descriptive summary
The homepage collection tiles include an image which has alt text. When the image IS NOT a thumbnail from a work, the alt text is "Collection" which is not very helpful. When the image IS a thumbnail from a work, the alt text is the work id which is disruptive and confusing.

The image is decorative since it is part of a link with descriptive text and provides no additional meaningful context. The alt text should be empty `alt=""`.

### Expected behavior
Alt text for collection images is `alt=""`

### Related work
Audit from #697

### Accessibility Concerns
Image alt text is vital for providing context and information about an image. These images provide no context or additional information. They are part of a link with sufficiently descriptive context and link text.

**Comment by raybrarian** [comment]
All decorative images on home page have blank `alt`tags. QA pass.
